ksqldb-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -258,6 +258,14 @@
       = "The maximum amount of pull query bandwidth in megabytes allowed over"
       + " a period of one hour. Once the limit is hit, queries will fail immediately";
 
+  public static final String KSQL_QUERY_PUSH_SCALABLE_MAX_HOURLY_BANDWIDTH_MEGABYTES_CONFIG
```

Comment:
We now changed the names of the spq config prefix.  It would be good to change it to v2.